.github/workflows/linux-build-base.yml

```
@@ -176,3 +176,78 @@ jobs:
       - name: Run Tests
         run: |
           cd _build/debug && ctest -j 8 --output-on-failure --no-tests=error
+
+  breeze-ubuntu-debug:
+    runs-on: 8-core-ubuntu
```

The build time is quite fast so we don't need the more expensive runner imo.
```suggestion
    runs-on: ubuntu-22.04
```

---

Done